WooCommerce/Classes/Extensions/TimeZone+Woo.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+
+/// TimeZone: Constant Helpers
+///
+extension TimeZone {
+
+    /// Returns the TimeZone using the timezone configured in the current website
+    ///
+    static var websiteTimezone: TimeZone {
+        guard let gmtOffset = ServiceLocator.stores.sessionManager.defaultSite?.gmtOffset else {
+            return .current
```

Comment:
not super sure if it makes sense, maybe if the GMT offset isn't available, we could fall back to the `timezone` field then `.current`?

Reply:
Unfortunately, the `timezone` property is not reliable (it works only with the country's timezone and not with UTC + offset), and we need to consider that in this specific case when `gmtOffset` is not available, also `timezone` is not available.
The only unfortunate case in which `gmtOffset` will not be available is the first time the app is opened with the website synchronization call that fails, or during the update in which we have introduced this property (and also here the API call need to fails).
So the best fallback seems to be `.current`, which will be the only available.